app/data/ukis_main.json

```diff
@@ -0,0 +1,5112 @@
+{
+    "data_version": "0.0.2",
+    "description": "UK Innovation Survey",
+    "questionnaire_id": "",
+    "schema_version": "0.0.1",
+    "title": "UK Innovation Survey",
+    "navigation": true,
+    "survey_id": "uk-innovation-survey",
```

Comment:
@benjefferies The intro page is implemented in [this PR](https://github.com/ONSdigital/eq-survey-runner/pull/831)